posthog/models/person/person.py

```diff
@@ -134,14 +134,6 @@ class Meta:
     uuid = models.UUIDField()
 
 
-class PendingPersonOverride(models.Model):
-    id = models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")
-    team_id = models.BigIntegerField()
-    old_person_id = models.UUIDField()
-    override_person_id = models.UUIDField()
-    oldest_event = models.DateTimeField()
-
-
```

Comment:
Very silly, but I just moved this from between `PersonOverrideMapping` and `PersonOverride` as those tables will no longer be needed after we move to the new flat table. In retrospect, this should have been before both or after both, it doesn't make sense for me to have put it in between them.